templates/ces_client/index.html

```diff
@@ -12,9 +12,14 @@
 
 <div class='row'>
     <div class='col-10 offset-1 mt-5 mb-3 pb-3'>
-        <h3>Connect to End Homelessness</h3>
-        <h4>A unified community response from the Corporation for Supportive Housing</h4>
-        <p>Are you looking for a place to stay tonight? Are you experiencing some form of homelessness? Talk to B.E.N. He will help guide you to your next location. Click "Goodbye!" or refresh the page.</p>
+        <div class='row'>
+            <div class='col'>
+                <a class="img-fluid float-left mr-3" href="{% url 'index' %}"><img src="{% static 'images/CESlogo.svg' %}" width='60'></a>
+                <h3>Connect to End Homelessness</h3>
+                <h4>A unified community response</h4>
+            </div>
+        </div>
+        <p>This site helps people experiencing homelessness find appropriate resources. If you need help, please talk to the bot B.E.N. in the chat window below. He will ask you questions and, when possible, recommend a phone number to call or address to go to. If you want to exit, then click "Goodbye".</p>
     </div>
```

Comment:
I would suggest refactoring this as two columns. I would also make the logo larger. I hacked with some inline styles to give an idea of what I mean:

<img width="1280" alt="screen shot 2019-01-08 at 2 44 15 pm" src="https://user-images.githubusercontent.com/6961258/50858461-1449b200-1356-11e9-8aad-4a567989f79e.png">
<img width="951" alt="screen shot 2019-01-08 at 2 44 04 pm" src="https://user-images.githubusercontent.com/6961258/50858459-1449b200-1356-11e9-847b-b862de7290a4.png">

With perhaps more margin above the paragraph text.